src/modules/common/components/dashboard/dashboard.js

```diff
@@ -52,15 +59,35 @@ const Dashboard = ({ wallet, t }) => {
         <div className={`${styles.main}`}>
           <div className={styles.subContainer}>
             {
-              isLoggedIn
-                ? <WalletDetails className={styles.marginFix} isWalletRoute={false} />
-                : null
+              !isEmpty(currentAccount) && (
+                <>
+                  <WalletDetails className={styles.marginFix} isWalletRoute={false} />
+                  <RecentTransactions
+                    className={styles.marginFix}
+                    isLoggedIn
+                  />
+                </>
+              )
+            }
+            {
+              isEmpty(currentAccount) && accounts.length === 0 && <AccountCreationTips />
+            }
+            {
+              isEmpty(currentAccount) && accounts.length > 0
```

Comment:
```suggestion
              isEmpty(currentAccount) && accounts.length
```

Reply:
So I applied this, and DeepScan threw:
> Numeric value 0 can be rendered because 'accounts.length' itself is checked. Consider checking 'accounts.length > 0' instead.